packages/app-harness/src/components/CastButton/index.android.mobile.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CastButton, useRemoteMediaClient } from 'react-native-google-cast';
+
+export function CastComponent() {
+    const client = useRemoteMediaClient();
+
+    if (client) {
```

Comment:
is this component re-renders for some side reason is having this in render cycle wont break anything? better to put it in useEffect?

Reply:
https://react-native-google-cast.github.io/docs/getting-started/usage
Fair enough, will update that, did it according to documentation example of usage